### DISCUSSION TITLE
How many known elements are in the periodic table today?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#821] [Galaxy Trivia] [answered]
**Bonus:** How many of them are hiding here?

<img width="462" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/fca85e63-f86b-45ec-95e2-43e8b3f28516">

**Comment by ParanoidUser** [comment]
@SpiffGreen / @MelvinAguilar, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
The periodic table has evolved to reflect over 150 years of scientific development and understanding in chemistry and physics. Today, it has **118 known elements**. It is widely regarded as one of the most significant achievements in science.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/34d13fc9-ff65-4cb2-aff6-3604ca17fedc)

**Reply by SANKARAMDAS** [reply]
Oh man(@peterbaccaro), you seriously did deep research on it...

**Reply by ParanoidUser** [reply]
Impressive! 👏 Did we miss my favorite *Ruthenium*?

**Reply by peterbaccaro** [reply]
> Impressive! 👏 Did we miss my favorite _Ruthenium_?

No, I can see Ruthenium, it's in yellow.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/75549f37-22f7-4257-9aba-efd517fafc2c)
